src/main/java/com/dife/api/service/ChatService.java

```diff
@@ -175,4 +185,68 @@ public Chat saveChat(Member member, Chatroom chatroom, String message) {
 		chatRepository.save(chat);
 		return chat;
 	}
+
+	public ChatResponseDto addChatFiles(
+			List<MultipartFile> chatFiles, Long chatroomId, String memberEmail)
+			throws JsonProcessingException {
+		Member member =
+				memberRepository.findByEmail(memberEmail).orElseThrow(MemberNotFoundException::new);
+
+		Chatroom chatroom =
+				chatroomRepository.findById(chatroomId).orElseThrow(ChatroomNotFoundException::new);
+
+		if (!chatroom.getMembers().contains(member))
+			throw new ChatroomException("채팅방 회원만이 채팅 파일을 보낼 수 있습니다!");
+
+		Chat chat = new Chat();
+		chat.setMember(member);
+		chat.setChatroom(chatroom);
+		chatroom.getChats().add(chat);
+
+		chatRepository.save(chat);
+
+		if (chatFiles != null && !chatFiles.isEmpty()) {
+			List<File> files =
+					chatFiles.stream()
+							.map(
+									file -> {
+										FileDto fileDto = fileService.upload(file);
+										File mappedFile = modelMapper.map(fileDto, File.class);
+										mappedFile.setChat(chat);
+										return mappedFile;
+									})
+							.collect(Collectors.toList());
+
+			chat.setFiles(files);
+			fileRepository.saveAll(files);
+		}
```

Comment:
가독성이 떨어지는 것 같아서 map안에 함수를 private 함수로 빼서 작성하는게 더 깔끔할 것 같네요! :) 
```suggestion
file -> {
함수
}
```